verifyBucketSproxydKeys.js

```diff
@@ -80,15 +86,15 @@ Optional environment variables:
     still done
 `;
 
-if (!BUCKETS && !RAFT_SESSIONS) {
-    console.error('ERROR: either BUCKETS or RAFT_SESSIONS environment '
+if (!BUCKETS && !RAFT_SESSIONS && !KEYS_FROM_STDIN) {
+    console.error('ERROR: either BUCKETS or RAFT_SESSIONS or KEYS_FROM_STDIN environment '
                   + 'variable must be defined');
     console.error(USAGE);
     process.exit(1);
 }
-if (BUCKETS && RAFT_SESSIONS) {
-    console.error('ERROR: BUCKETS and RAFT_SESSIONS environment variables '
-                  + 'cannot be both defined');
+if ((BUCKETS ? 1 : 0) + (RAFT_SESSIONS ? 1 : 0) + (KEYS_FROM_STDIN ? 1 : 0) > 1) {
```

Comment:
A little bit of a stretch again...

Instead of simply checking if the value is undefined or not what about using what @tmacro did with converting a string to a boolean. With values such as `true,false,1,0...`. This would allow using true/false as opposed to exists or not. It is kinda one or nothing so it might not make sense but maybe if I had an automated tool where it put in true/false as opposed to include or not include it might matter.

Then with 3 booleans, you can just add them up.
JS will let you add 3 booleans and its treated as 0/1.
```suggestion
if (BUCKETS + RAFT_SESSIONS + KEYS_FROM_STDIN > 1) {
```
You can also combine this with the previous check and simply do `!=1` as the error messages are about the same.